src/Clicksign/ClientBase.php

```diff
@@ -79,6 +85,7 @@ protected function doRequest($url, $method, $data, $contentType = null)
         curl_setopt($c, CURLOPT_URL, $url);
         curl_setopt($c, CURLOPT_SSL_VERIFYHOST, 0);
         curl_setopt($c, CURLOPT_SSL_VERIFYPEER, 0);
+        curl_setopt($c, CURLOPT_PROXY, $this->proxy);
```

Comment:
@hugosbg desculpa a pergunta básica, confesso que PHP não é bem a minha praia, o que acontece se `$this-proxy` não for setado? A chamada `curl_setopt` com a opção `CURLOPT_PROXY` está disponível em quais versões do PHP?